package.json

```diff
@@ -62,6 +62,7 @@
     "ismobilejs": "^0.4.0",
     "object-assign": "^4.0.1",
     "pixi-gl-core": "^1.0.3",
+    "remove-array-items": "^0.0.1",
```

Comment:
Would you mind bumping **remove-array-items** to version 1.0.0 and publishing? This would be important for a couple of reasons:
* this utility is likely to change in the near future and is stable enough
* using full semver gives us more information about what changes into the future
* we're not a fan of pre-v1 versioning
* the semantic constraint `^0.0.1` doesn't resolve to anything other than `0.0.1` (for instance, 0.0.2 does not satisfy constraint ^0.0.1, [check here](http://jubianchi.github.io/semver-check/)). So we would never be able to pick up future changes